src/test/feature/ManagementTool/test_hawq_register.cpp

```diff
@@ -199,9 +199,121 @@ TEST_F(TestHawqRegister, TestNotHDFSPath) {
 	util.execute("create table hawqregister(i int);");
 	util.query("select * from hawqregister;", 0);
 
-	EXPECT_EQ(1, Command::getCommandStatus("hawq register " + (string) HAWQ_DB + "hawqregister /hawq_register_hawq.paq"));
+	EXPECT_EQ(1, Command::getCommandStatus("hawq register -d " + (string) HAWQ_DB + " -f /hawq_register_hawq.paq hawqregister"));
 	util.query("select * from hawqregister;", 0);
 
 	EXPECT_EQ(0, Command::getCommandStatus("hadoop fs -rm hdfs://localhost:8020/hawq_register_hawq.paq"));
 	util.execute("drop table hawqregister;");
 }
+
+TEST_F(TestHawqRegister, TestUsage1ParquetRandomly) {
+  SQLUtility util;
+  string rootPath(util.getTestRootPath());
+  string relativePath("/ManagementTool/test_hawq_register_hawq.paq");
+  string filePath = rootPath + relativePath;
+  EXPECT_EQ(0, Command::getCommandStatus("hadoop fs -put -f " + filePath + " hdfs://localhost:8020/hawq_register_hawq.paq"));
+  util.execute("drop table if exists nt;");
+  util.execute("create table nt(i int) with (appendonly=true, orientation=parquet);");
+  EXPECT_EQ(0, Command::getCommandStatus("hawq register -d " + (string) HAWQ_DB + " -f hdfs://localhost:8020/hawq_register_hawq.paq nt"));
+	util.query("select * from nt;", 3);
+	util.execute("insert into nt values(1);");
+	util.query("select * from nt;", 4);
+  util.execute("drop table nt;");
+}
+
+TEST_F(TestHawqRegister, TestUsage1ParquetRandomly2) {
+  SQLUtility util;
```

Comment:
@xunzhang What's the difference between these two test cases: TestUsage1ParquetRandomly and TestUsage1ParquetRandomly2?